openscad/README.md

```diff
@@ -0,0 +1,70 @@
+# OpenScad COVID-19 v18 Stacks
+
+This directory contains an [OpenScad](https://www.openscad.org/) script for generating stacks of arbitrary heights of the COVID-19 v18 face shield model.
+
+![Example stack of 10 masks](stack_of_10.png)
+
+## Usage
+
+*Instructions here worked on Ubuntu 18.04, but may need to be slightly altered for other operating systems.*
+
+In order to use the script you'll first need to install [OpenScad](https://www.openscad.org/) (further instructions can be found [here](http://ubuntuhandbook.org/index.php/2019/01/install-openscad-ubuntu-18-10-18-04/). 
+
+```
+sudo add-apt-repository ppa:openscad/releases
+sudo apt-get update
+sudo apt-get install openscad
+```
+
+Once you've run the above commands you should have `openscad` available to you via command line:
+
+```
+chris@ubuntu:~/git/face-shield$ which openscad
+/usr/local/bin/openscad
+```
+
+You'll now be able to run the accompanying script found in this directory (`covid_19.scad`) to generate a stack of an arbitrary height. Note that rendering time is dependent upon how tall your make the stack, so taller stacks will take longer to generate. In my experience, a stack of 14 face shields took around 60 seconds to finish computing on a fairly beefy machine.
+
+The following command generates a stack of height `6` and saves the output to a file `covid_19_6_stack_v18.stl`:
+
+```
+openscad -o covid_19_6_stack_v18.stl covid_19.scad -D "shield_stack(count = 6)"
+```
+
+## Printing
+
+I have successfully printed stacks of up to 14 on a Creality CR-10s using a 1.0mm nozzle. With the profile found in the `profiles/creality/foobar` file the printing process for two stacks of 14 on a single bed is 23 hours and takes up 990g of material.
+
+Note that in the default configuration for the model the separators between shields are `0.5mm` tall, so when you're printing your layer height should be `0.5mm` max or you'll risk having the shields printed in a way where they cannot be separated.
+
+If you're using something other than the CR-10s, here are the general parameters I've had success with here:
+
+* **PLA** - Haven't tried PETG but I hear it works well, these settings probably won't work for it.
```

Comment:
All my tests with stacking PETG failed because the layers bond too strongly together. So I would point out that PLA is strictly recommended for stacks.